docs/modules/operation/pages/database-reports/database.adoc

```diff
@@ -235,16 +240,16 @@ The following example shows the use of that helper.
 <source aggregation="AVERAGE" label="IfOutOctets" attribute="ifHCOutOctets" transient="false" resourceId="$P{nodeResourceDescriptor}.interfaceSnmp[en0-005e607e9e00]"/>
 </query-request>]]>
 ------
-Depending on the input parameters, you either get a `node resource descriptor` or a `foreign source/foreign id resource descriptor`.
+Depending on the input parameters, you get either a node resource descriptor or a foreign source/foreign id resource descriptor.
 
 [[ga-database-reports-interface-descriptor]]
 === Interface descriptor use
 
-An `interfaceSnmp` is addressed with the exact `interface descriptor`.
-To allow easy access to the `interface descriptor` we provide a helper tool.
+An interfaceSnmp is addressed with the exact interface descriptor.
+To allow easy access to the interface descriptor, we provide a helper tool.
 The following example shows the use of that helper.
 
-.jrxml report snippet to visualize the use of the `interface descriptor`
+.jrxml report snippet to visualize the use of the interface descriptor
```

Comment:
There are two `\` in the following source block that can be removed.

Reply:
I see 5. Can I remove all of them?